pkg/middlewares/replicate/replicate.go

```diff
@@ -38,13 +39,13 @@ type replicate struct {
 
 // New creates a new http handler.
 func New(ctx context.Context, next http.Handler, config *runtime.MiddlewareInfo, middlewareName string) http.Handler {
-	ctx = middlewares.GetLoggerCtx(ctx, middlewareName, typeName)
-	log.FromContext(ctx).Debug("Creating middleware")
+	ctx = middlewares.GetLoggerCtx(ctx, middlewareName, middlewareType)
+	logger := log.FromContext(ctx)
 
+	logger.Debugf("Creating middleware %v", middlewareName)
```

Comment:
inconsistent with the other messages format. Suggestion: start with the lower case if possible, i.e. "creating middleware %v".

Reply:
fixed